web/src/features/landingPage/pages/LandingPage.tsx

```diff
@@ -1,68 +1,79 @@
 import React, { useEffect } from 'react'
-import { useTheme } from '@mui/material/styles'
+import { useTheme, styled } from '@mui/material/styles'
+import { theme } from 'app/theme'
 import useMediaQuery from '@mui/material/useMediaQuery'
-import makeStyles from '@mui/styles/makeStyles'
 import ContentContainer from 'app/ContentContainer'
 import Footer from 'features/landingPage/components/Footer'
 import Sidebar from 'features/landingPage/components/Sidebar'
 import ToolCards from 'features/landingPage/components/ToolCards'
 import { LANDING_PAGE_DOC_TITLE } from 'utils/constants'
 
-const useStyles = makeStyles(theme => ({
-  content: {
-    display: 'flex',
-    flexGrow: 1
-  },
-  root: {
-    display: 'flex',
-    flexDirection: 'column',
-    minHeight: '100vh'
-  },
-  subcontainer: {
-    display: 'flex',
-    flexDirection: 'column',
-    flexGrow: 1
-  },
-  sidebar: {
-    display: 'flex',
-    flexGrow: 1,
-    overflowY: 'auto',
-    [theme.breakpoints.up('sm')]: {
-      minWidth: '251px',
-      maxWidth: '251px',
-      width: '251px'
-    }
+const PREFIX = 'LandingPage'
+
+const Root = styled('div', {
+  name: `${PREFIX}-root`
+})({
+  display: 'flex',
+  flexDirection: 'column',
+  minHeight: '100vh'
+})
+
+const Content = styled('div', {
+  name: `${PREFIX}-content`
+})({
+  display: 'flex',
+  flexGrow: 1
+})
+
+const SubContainer = styled('div', {
+  name: `${PREFIX}-subContainer`
+})({
+  display: 'flex',
+  flexDirection: 'column',
+  flexGrow: 1
+})
+
+const SidebarContainer = styled('div', {
+  name: `${PREFIX}-sidebarContainer`
+})({
+  display: 'flex',
+  flexGrow: 1,
+  overflowY: 'auto',
+  [theme.breakpoints.up('sm')]: {
+    minWidth: '260px',
```

Comment:
This css seems a little odd...especially specifying a minWidth that is wider than the width property? Also, these values don't match the old ones?

Reply:
Yeah.. weird, must have been some inbetween code migration where it was looking weird that I forgot to go back to fix, fixed in: https://github.com/bcgov/wps/pull/2998/commits/7131d59909d5b7b6bc815ac03802213c135749f4